instance/tests/integration/integration_instance.py

```diff
@@ -489,12 +490,22 @@ def test_openstack_server_terminated(self):
         """
         Test that OpenStackServer detects if the VM was terminated externally.
         """
-        server = OpenStackServer(name_prefix="integration_test")
-        server.save()
-        server.start()
-        server.sleep_until(lambda: server.status.accepts_ssh_commands, timeout=120)
-        server.os_server.delete()
-        self.assert_server_terminated(server)
+        retries = 1
+        while retries >= 0:
```

Comment:
Can we move this code into a helper function? Call it `spawn_openstack_server` and make it take an optional `retries` parameter, which defaults to 1?

This test will then look like this:
```python
server = spawn_openstack_server()
server.os_server.delete()
self.assert_server_terminated(server)
```